oak_abi/src/label/mod.rs

```diff
@@ -66,8 +66,18 @@ impl crate::proto::oak::label::Label {
         // The target label must have (compared to the self label):
         // - same or more confidentiality tags
         // - same or fewer integrity tags
-        self_confidentiality_tags.is_subset(&other_confidentiality_tags)
-            && other_integrity_tags.is_subset(&self_integrity_tags)
+        (other_confidentiality_tags.contains(&Tag::universal_top_tag())
+            || self_confidentiality_tags.is_subset(&other_confidentiality_tags))
+            && (self_integrity_tags.contains(&Tag::universal_top_tag())
+                || other_integrity_tags.is_subset(&self_integrity_tags))
+    }
+}
+
+impl Tag {
+    /// Convenience function for creating top privilege tag that can declassify all types of tags.
```

Comment:
I think this comment mixes privilege and tags a bit, AFAICT it is just creating an instance of the top tag. Also I would just call it top, universal_top seems just another name for the same thing, and it is already called top elsewhere (and in literature)

Reply:
I included `universal` in the name to distinguish it from the case where we'll have separate tops for each sub-lattice. I have changed it to `top` for now.